sonar-db/src/main/java/org/sonar/db/permission/PermissionTemplateDto.java

```diff
@@ -66,11 +66,26 @@ public String getKee() {
     return kee;
   }
 
+  /**
+   * @since 5.2 the kee column is a proper uuid. Before that it was build on the name + timestamp
+   */
+  public String getUuid() {
```

Comment:
As discussed the db column must be renamed in order to remove the ambiguity. Key and uuid in the DTO are the same.